File: chroma-league/src/test/java/net/booone/chromaleague/hud/animations/EnemyElderDragonKillAnimationTest.java
package net.booone.chromaleague.hud.animations;

import net.booone.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyElderDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyElderDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyElderDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyElderDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyElderDragonKillAnimation} from '../../../src/hud/animations/EnemyElderDragonKillAnimation.js';

/**
 * Integration test for enemy elder dragon kill animation on actual hardware.
 */
describe('EnemyElderDragonKillAnimation Integration', () => {
    test('playEnemyElderDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyElderDragonKillAnimation());
    }, 30000);
});
